pkg/server/api_plot.go

```diff
@@ -357,7 +357,9 @@ func makePlotsResponse(plotSeries map[string][]plot.Series, plotReq *PlotRequest
 					plotItem.Scale(plot.Value(scale))
 				}
 
-				seriesOptions[seriesItem.Name] = seriesItem.Options
+				if len(seriesItem.Options) != 0 {
```

Comment:
Sorry I didn't receive a notification for your reply.

Indeed, I understand that having to fill units every time when you create graphs could be a little bit repetitive.

I guess we can implement some metrics defaults in providers configuration files as for filters using patterns and setting default values for units and/or scales. Then, leaving a possibility to override them in the UI when creating a graph.

For example:
```
…
"metrics_defaults": [
    { "pattern": "net\..+\.octets", "unit": "bits", "scale": 0.125 }
]
```

WDYT?